_guides/analytics/analytics_basics.md

```diff
@@ -0,0 +1,229 @@
+it may not fire.
+
+### Simple amp-analytics configuration
+
+To create a simple
+[amp-analytics](/docs/reference/extended/amp-analytics.html) configuration,
+you must first include this `custom-element` declaration
+in the head of the AMP document:
+
+{% highlight html linenos %}
+<script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
+{% endhighlight %}
+
+Similar to the [amp-pixel example](/docs/guides/analytics/analytics_basics.html#simple-amp-pixel-configuration),
+the following `amp-analytics` sample tracks page views and
+sends the data to a defined URL along with a random ID: 
+
+{% highlight html linenos %}
+<amp-analytics>
+<script type="application/json">
+{
+  "requests": {
+    "pageview": "https://foo.com/pixel?RANDOM",
+  },
+  "triggers": {
+    "pixel emulation": {
+      "on": "visible",
+      "request": "pageview"
+    }
+  }
+}
+</script>
+</amp-analytics>
+{% endhighlight %}
+
+The `requests` attribute defines what data gets sent to where.
+In this example, the data is the `pageview`, and it’s sent
+to the specified URL, `https://foo.com/pixel?`, along with a random ID.
+
+The `triggers` attribute defines when the data gets sent.
+When the page is `visible`,
+an event triggers and the `pageview` request is sent.
+Learn more about [requests and triggers](/docs/guides/analytics/deep_dive_analytics.html#requests-triggers--transports).
+
+## Variable substitution
+
+Both the [amp-pixel](/docs/reference/amp-pixel.html) and
+[amp-analytics](/docs/reference/extended/amp-analytics.html) components
+allow all standard URL variable substitutions (see also
+[AMP HTML Variable Substitutions](https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md)).
+In the following example,
+the page view request is sent to the URL,
+along with the current AMP document’s canonical URL, it’s title, and a
+[client ID](/docs/guides/analytics/analytics_basics.html#user-identification):
+
+{% highlight html linenos %}
+<amp-pixel src="https://example.com/analytics?url=${canonicalUrl}&title=${title}&clientId=${clientId(cid-scope)}"></amp-pixel>
+{% endhighlight %}
+
+Due to it’s simplicity,
+the `amp-pixel` tag can only include variables defined by the platform
+or that the AMP runtime can parse from the AMP page.
+The `amp-analytics` tag can include the same variables as `amp-pixel`,
+as well as uniquely defined variables inside the tag configuration. 
+
+Use the format `${varName}` in a request string for a page
+or platform-defined variable.
+The `amp-analytics` tag will replace the template with its actual value
+at the time of construction of the analytics request (see also
+[Variables supported in amp-analytics](https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/analytics-vars.md)).
+
+In the following example, the page view request is sent to the URL,
+with additional data extracted from variable substitutions,
```

Comment:
which variables do you see as being provided by variable substitutions vs. the platform? I think we should talk about this in terms of two sources of definition: the platform and variable from the config itself. It would also be good to parenthetically reference which cases match which variables:

- platform: canonicalUrl, clientId(site-user-id)
- inline defined: title, account